doc/content/integrations/webhooks/cloud-integrations/tagoio/tts-setup/_index.md

```diff
@@ -12,14 +12,14 @@ Next, create a Webhook integration on {{% tts %}} by following this section.
 
 Fill in the **Webhook ID** field. 
 
-Create a `Content-Type` header entry with `application/json` value, and a `Device-Token` header entry with the token value you previously copied from TagoIO.
+Create an `Authorization` header entry with the authorization copied in the previous step as a value.
```

Comment:
Shouldn't we drop this completely and point this to the template ?

Reply:
Sure, I can do it, it makes sense now when we have webhook templates set up. @laurensslats can you confirm if I should update all the integration guides to point to webhook templates now and remove this kind of info that's basically hidden from the user during creation?

Reply:
@adriansmares just for the record, we have decided to keep this, but to mention that there is a webhook template available in the beginning of the doc